pkg/apis/core/v1alpha1/federatedtypeconfig_types.go

```diff
@@ -119,11 +119,13 @@ func setStringDefault(value *string, defaultValue string) {
 	*value = defaultValue
 }
 
-// PluralNameForKind naively computes the plural name from the kind by
-// lowercasing and suffixing with 's'.
-func pluralName(kind string) string {
+// PluralName computes the plural name from the kind by
+// lowercasing and suffixing with 's' or `es`.
```

Comment:
@danehans do we need to handle the case if the word ends up with `y`? Such as `library` should be `libraries`?

Reply:
Probably a good idea.

Reply:
I call YAGNI - let's focus on stuff we actually have a need for.  I think it's premature to optimize any one piece while we're still in alpha.

Reply:
:-)